packages/suite/src/types/wallet/fiatRates.ts

```diff
@@ -1,27 +1,7 @@
-export interface TickerId {
-    symbol: string;
-    mainNetworkSymbol?: string; // symbol of thee main network. (used for tokens)
-    tokenAddress?: string;
-}
-
-export interface CurrentFiatRates {
-    symbol: string;
-    rates: { [key: string]: number | undefined };
-    ts: number;
-}
-
-export interface TimestampedRates {
-    rates: { [key: string]: number | undefined };
-    ts: number;
-}
-
-export interface LastWeekRates {
-    symbol: string;
-    tickers: TimestampedRates[];
-    ts: number;
-}
-
-export interface CoinFiatRates extends TickerId {
-    current?: CurrentFiatRates;
-    lastWeek?: LastWeekRates;
-}
+export type {
```

Comment:
Why not import directly from `@suite-common/wallet-types` where needed?

Reply:
Reexports will be removed in follow-up, to reduce big diff